build.gradle

```diff
@@ -38,7 +38,7 @@ allprojects {
     implementation group: 'com.google.guava', name: 'guava', version: guava_version
     implementation group: 'com.typesafe', name: 'config', version: typesafe_config_version
     implementation group: 'io.netty', name: 'netty-all', version: netty_version
-    implementation group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: netty_tcnative_version, classifier: nativeClassifier
+    implementation group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: netty_tcnative_version
```

Comment:
can you remove the nativeClassifier determination code as well?

Reply:
fixed